### PR TITLE
Site Settings: Delete Atomic sites

### PR DESCRIPTION
**Pull request by Copons**
Fixes 290-gh-automated-transfer

Add the Delete Site flow to Atomic sites.
As there isn't an automated Atomic sites deletion, instead of showing a "Delete Site" button, we prompt the user to contact support.

This also improves the copy of both the Site Tools and the Delete Site pages.

### Screenshots

| Simple Site | Atomic Site |
| --- | --- |
| screen shot 2017-11-09 at 11 34 11 | screen shot 2017-11-09 at 11 34 44 |
| screen shot 2017-11-09 at 11 33 59 | screen shot 2017-11-09 at 11 34 30 |

### CODE REVIEW

**Comment by apeatling** [comment]
@jhnstn / @donpark / @retrofox Can we get a review and a resolution on this one? Thanks.

**Comment by apeatling** [comment]
@Copons Will need a rebase first.

**Comment by Copons** [comment]
PR rebased and ready for a last 👀!

**Comment by apeatling** [comment]
Looking at this one more time, can we add a line that explains that they need to contact support to delete their site? Either that or make the call to action something like "Contact Support to Delete your Site" (seems too long).

I think the action of contacting support is disconnected from the expected action of deleting your site, so it needs an explanation line.

**Comment by apeatling** [comment]
What about making the first line of explanation text say. "You will need to contact support to delete your site." ... and then the rest.

**Comment by gwwar** [comment]
Did we verify what happens to the related domain? I believe the WordPress subdomain is blacklisted for future use, but if folks have purchased a domain they should be able to transfer/reuse that.

**Comment by gwwar** [comment]
Testing it looks like that's we have a similar domain issue for simple sites. Overall this tests well, I think we should be able to 🚢 after you're happy with copy changes.

**Comment by michelleweber** [comment]
I have a few small suggestions to smooth out/fix little issues on all of these screens, and make the tone a little warmer :)

Site tools:
- Export content from your site. You own your data -- take it anywhere! OR
- Export content from your site.
- Delete all your posts, pages, media, and data, and give up your site's address.

(In "export content," either stick with the basic detail, or expand on "own your data" -- it feels a little fragmented as-is. In "delete all your posts...." you need a comma after "media.")

Delete site, Atomic:
- Before deleting your site, take a moment to export your content. This will package the content of all your posts and pages, along with your site's settings, into a .zip file. If you ever want to re-create your site, you'll be able to import the .zip file to a new site. This content can not be recovered once your delete this site.
- To delete this site, you'll need to contact our support team. Deletion can not be undone, and will remove all content, contributors, domains, and upgrades from this site. If you're unsure about what deletion means or have any other questions, you'll have a chance to chat with someone from our support team before anything happens. 

Delete site, simple:
- Deletion can not be undone, and will remove all content, contributors, domains, and upgrades from this site. If you're unsure about what deletion means or have any other questions, please chat with someone from our support team before proceeding. 

Do simple sites also get the "please export" note? (If not, why not?)

**Comment by Copons** [comment]
Thanks @michelleweber!

> (In "export content," either stick with the basic detail, or expand on "own your data" -- it feels a little fragmented as-is. In "delete all your posts...." you need a comma after "media.")

I like the "you own your data" bit.
I'll go with: "Export content from your site. You own your data — take it anywhere!" (hopefully the em dash won't screw up anything...)

> Do simple sites also get the "please export" note? (If not, why not?)

They do, I've just not included it in the Simple site screenshot because it wasn't relevant to this PR change.

File: client/my-sites/site-settings/site-tools/index.jsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ class SiteTools extends Component {
 		);
 		const deleteSite = translate( 'Delete your site permanently' );
 		const deleteSiteText = translate(
-			'Delete all your posts, pages, media and data, ' + "and give up your site's address."
+			"Delete all your posts, pages, media, and data, and give up your site's address."
```
Comment (a8ci18n) [inline client/my-sites/site-settings/site-tools/index.jsx]: Hi! I've found a possible matching string that has already been translated **25** times:
`translate( 'Delete all your posts, pages, media and data, and give up your site's address.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by Copons** [comment]
Screenshots in the PR description updated according to the new copy.

**Comment by westi** [comment]
Is there a reason this button doesn't pre-populate the form for the user?  Or try and Launch live chat?

**Pull request merged**